src/EditBaniOrder/EditBaniOrder.jsx

```diff
@@ -42,11 +55,40 @@ function EditBaniOrder({ navigation }) {
   );
 
   useEffect(() => {
-    batch(() => {
-      dispatch(setBaniList(baniListdata));
-      dispatch(setBaniOrder({ baniOrder: orderData }));
-    });
-  }, [baniListdata, orderData]);
+    if (isReset) {
```

Comment:
Minor change: I have seen it's more prefferred to exit useEffect early by using a negative condition here.
```javascript
useEffect(() => {
   if (!isReset) return;

   // rest of the code here
```